services/src/test/java/edu/unc/lib/dl/cdr/services/processing/SetAsPrimaryObjectServiceTest.java

```diff
@@ -0,0 +1,144 @@
+/**
```

Comment:
I'm not sure why this file is showing up as new?  Maybe rebasing your branch on fcrepo4 will resolve this?

Reply:
??

i will try rebasing